### ISSUE TITLE
Weird UI bugs

### ISSUE THREAD

**Issue by negasora** [Vector35/snippets#35]
- Color formatting looks like it's early by a character
- Line numbers start at 3
![image](https://user-images.githubusercontent.com/3020965/234675757-61dcad38-7e57-4048-ac4b-2b5bfe36088f.png)

**Comment by psifertex** [comment]
Yeah, this is... known. The short answer is don't have empty lines up top, I haven't figured out how to fix it otherwise.

**Comment by psifertex** [comment]
The line numbers are intentional. The first two lines of the file are actually metadata with description and hotkey. This makes it so that errors are actually corresponding to the right number.
